README.md

```diff
@@ -1528,6 +1530,27 @@ Placeholders `[STATUS]` and `[BODY]` as well as the fields `endpoints[].body`, `
 
 This works for SCTP based application.
 
+### Monitoring a WebSocket endpoint
+By prefixing `endpoints[].url` with `ws://` or `wss://`, you can monitor WebSocket endpoints at a very basic level:
+
+```yaml
+endpoints:
+  - name: example
+    url: "wss://example.com/"
+    body: "status"
+    jsonrpc: true
+    conditions:
+      - "[CONNECTED] == true"
+      - "[BODY].result >= 0"
+```
+
+The `jsonrpc` flag specifies whether to wrap the request body in a JSON RPC 2.0
+method call, in which the first word becomes method name and the rest becomes
+parameters. For example: `{{"jsonrpc":"2.0","method":"status","params":[],"id":1}}`
```

Comment:
I'm not very familiar with websockets - would all websocket endpoints be expected to have `jsonrpc` set to `true`?

Reply:
@TwiN, I removed the `jsonrpc` from the example in the README and updated the text with a hopefully better explanation.

Let me know what are your thoughts about it.